hazelcast/src/test/java/com/hazelcast/jet/impl/connector/WriteJdbcPTest.java

```diff
@@ -101,19 +102,49 @@ public static void setupClass() throws SQLException {
 
     @Before
     public void setup() throws SQLException {
-        tableName = "T" + tableCounter.incrementAndGet();
+        tableName = "T" + TABLE_COUNTER.incrementAndGet();
         logger.info("Will use table: " + tableName);
-        try (Connection connection = ((DataSource) createDataSource(false)).getConnection()) {
-            connection.createStatement()
-                    .execute("CREATE TABLE " + tableName + "(id int, name varchar(255))");
-        }
+        /* language=SQL */
+        executeSql("CREATE TABLE " + tableName + "(id int, name varchar(255))");
     }
 
     @After
     public void tearDown() throws Exception {
         if (hikariDataSource != null) {
             hikariDataSource.close();
         }
+
+        listRemainingConnections();
+        // kill any hanging connection
+        /* language=SQL */
+        executeSql("SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE pid <> pg_backend_pid()");
+    }
+
+    private static void executeSql(String sql) throws SQLException {
```

Comment:
I actually meant the following, then it is in single place:

```suggestion
    private static void executeSql(/* language=SQL */ String sql) throws SQLException {
```